react-backend/app.js

```diff
@@ -32,7 +32,7 @@ app.use(cookieParser());
 app.use(express.static(path.join(__dirname, 'client/build')));
 
 app.use('/', index);
-app.use('/users', users);
+// app.use('/users', users);
```

Comment:
is this meant to be commented out?

Reply:
removed this line and the file as we never used it